CONTRIBUTING.md

```diff
@@ -26,7 +26,7 @@ instructions:
 ### Quick start
 
 1. fork the repo and clone it locally
-2. `npm run get-me-going` to install and build dependencies
+2. `npm run setup-workspace` to install and build dependencies
```

Comment:
This wording is a bit more clear I think

Reply:
Agreed, naming things is hard ™

Thanks for coming up with this!